### PR TITLE
Add name property for DacFx wizard edit text boxes

### PR DESCRIPTION
**Pull request by kisantia**
Adding name properties to file location edits, version edit, and target database edit requested in #6731. Also adds aria label for drop down component.
image

image

### CODE REVIEW

File: extensions/dacpac/src/wizard/api/dacFxConfigPage.ts [hunk 1]
```diff
@@ -149,6 +150,7 @@ export abstract class DacFxConfigPage extends BasePage {
 				required: true
 			}).component();
 
+		this.fileTextBox.ariaLabel = localize('dacfx.fileLocationEdit', "File Location Edit");
 		this.fileButton = this.view.modelBuilder.button().withProperties({
 			label: '•••',
```
Comment (Benjin) [inline extensions/dacpac/src/wizard/api/dacFxConfigPage.ts]: Separate, but this button probably needs an ariaLabel too

Comment (kisantia) [reply extensions/dacpac/src/wizard/api/dacFxConfigPage.ts]: yup there's an issue for that #6742. Will fix in a separate PR

File: extensions/dacpac/src/wizard/pages/extractConfigPage.ts [hunk 2]
```diff
@@ -111,8 +111,9 @@ export class ExtractConfigPage extends DacFxConfigPage {
 			required: true
 		}).component();
 
-		// default filepath
+		// default version
 		this.versionTextBox.value = '1.0.0.0';
+		this.versionTextBox.ariaLabel = localize('dacfx.versionEdit', "Version Edit");
```
Comment (udeeshagautam) [inline extensions/dacpac/src/wizard/pages/extractConfigPage.ts]: do you need to say **edit** - I think NVDA adds edit. Just check one that there is not "edit edit"

Comment (kisantia) [reply extensions/dacpac/src/wizard/pages/extractConfigPage.ts]: yeah I wasn't sure about that too. The expected text in the issue says edit. Should we ask for clarification on this?

image

**Pull request merged**